src/field_theory/adjoin.lean

```diff
@@ -356,10 +359,10 @@ by { conv_rhs { rw ← adjoin_simple.algebra_map_gen F α },
      rw is_integral_algebra_map_iff (algebra_map F⟮α⟯ E).injective,
      apply_instance }
 
-lemma adjoin_simple_adjoin_simple (β : E) : ↑F⟮α⟯⟮β⟯ = F⟮α, β⟯ :=
+lemma adjoin_simple_adjoin_simple (β : E) : F⟮α⟯⟮β⟯.restrict_scalars F = F⟮α, β⟯ :=
```

Comment:
I wasn't sure where to leave this comment, so I'm choosing a random example.

Is `F⟮α⟯⟮β⟯.restrict_scalars F` now the simp normal form? I missed whether there was a simp lemma that rewrote `↑F⟮α⟯⟮β⟯` into a `restrict_scalars` expression.

Reply:
The coercion form no longer exists, for consistency with `submodule.restrict_scalars` and `subalgebra.restrict_scalars`.